docs/clients/javascript/http.md

```diff
@@ -0,0 +1,95 @@
+Open your console and install the JS HTTP Client using npm.
+
+``` bash
+npm install @ceramicnetwork/http-client
+```
+
+### 2. Import the HTTP client
+
+``` javascript
+import CeramicClient from '@ceramicnetwork/http-client'
+```
+
+### 3. Configure your node URL
+
+``` javascript
+const API_URL = "https://yourceramicnode.com"
+```
+
+Available options for your node setup:
+
+- [Free community nodes](../../tools/hosted-nodes/community-nodes.md): Discover free HTTP endpoints
+- [Commercial node providers](../../tools/hosted-nodes/node-providers.md): Discover paid node providers
+- [Host your own node](../../run/nodes.md): Learn how to setup and host your own node
+- *localhost*: You may want to use the JS HTTP Client with a Ceramic node running on your local machine for development and testing. To achieve this, first start a local daemon by [installing the CLI](./cli.md). Once the CLI is installed, you can pass `https://localhost:7007` to the HTTP client. This setup will allow you to read streams from other nodes connected on the Ceramic network, but writes to your local node will only be available on nodes found on the [`peerlist`](https://github.com/ceramicnetwork/peerlist/blob/main/testnet-clay.json){:target="_blank"}. For now, these streams will not be available to other nodes on the network due to a limitation in `js-ipfs` which will be fixed in the future.
+
+### 4. Create a Ceramic instance
+
+``` javascript
+const ceramic = new CeramicClient(API_URL)
+```
+
+### 5. Import DID resolvers
+Import resolvers for all DID methods that will perform [writes](../../build/writes.md) using this HTTP Client. This should include all DID Methods that your app will use for [authentication](../../build/authentication.md). If your HTTP Client will only perform queries, then jump straight to the [queries](../../build/queries.md) page.
+
+``` javascript
+import KeyDidResolver from 'key-did-resolver'
+import ThreeIdResolver from '@ceramicnetwork/3id-did-resolver'
+```
+
+### 6. Create a resolver instance
+This should include all DID resolvers included in the previous step.
+
+``` javascript
+const resolver = { ...KeyDidResolver.getResolver(),
+                   ...ThreeIdResolver.getResolver(ceramic) }
+```
+
+### 7. Create a DID instance
+The DID instance wraps the resolver and also should include the DID Provider for the DID Method your users will use to [authenticate](../../build/authentication.md) to perform [writes](../../build/writes.md).
+
+``` javascript
+import { DID } from 'dids'
+const did = new DID({ resolver })
+```
+
+### 8. Set DID instance on HTTP client
+
+``` javascript
+ceramic.did = did
+```
+
+## **Next steps**
+After setting the DID instance on the HTTP client, your application will be able to support writes. Proceed to [setting up authentication](../../build/authentication.md) so users can authenticate to perform these writes. If your app only needs to perform queries, then jump ahead to [queries](../../build/queries.md).
```

Comment:
it's a little confusing since they won't really be able to perform writes until setting the did provider which is handled in the authentication page

Reply:
Fixed